Runtime/Serialization/NewtonsoftJsonSerializer/NewtonsoftJsonCourseSerializer.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
+using Boo.Lang.Runtime;
```

Comment:
🤔

Reply:
Is this ignored on purpose?

Reply:
pushed the change minutes after the rereview request, but it's not in there anymore